packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/add/add.html

```diff
@@ -4,6 +4,7 @@
 
 <div data-ng-if="!$ctrl.loading">
     <oui-back-button
+        data-on-click="$ctrl.goBack()"
         data-state="telecom.telephony.billingAccount.alias.configuration.lines"
```

Comment:
Maybe we can remove the `data-state` in that case ? It would also be interesting to know why `data-state` isn't working in that case, is the state valid ?

Reply:
I tried to remove it and let just the `data-on-click="$ctrl.goBack()"` but it's not working anymore.
I don't understand why is not working while we use same sample in many others place on the project: `<oui-back-button data-on-click="$ctrl.goBack()"`

Reply:
it's works fine if i move the `$ctrl.goBack()` on the span like:
`     <oui-back-button>
        <span data-ng-click="$ctrl.goBack()"
            data-translate="telephony_alias_config_contactCenterSolution_lines_add_line"
        ></span>
    </oui-back-button>`

Reply:
@ovh/team-manager do you know why i had this issue ?

Reply:
It looks like `goBack` method is not injected through component/controller, you might need to pass it so binding is done correctly